assets/css/style.css

```diff
@@ -85,3 +85,12 @@ footer {
     width: 300px;
     height: 600px;
   }
+
+table, tr, td, th {
```

Comment:
I assumed it would be a bit redundant seeing as the table is only ever inside of a post body. Then again, things could always change. Will take this under advisement and push an update.